languages/en/filters.genie

```diff
@@ -180,3 +181,218 @@ range_with_filter = {
         ]);
     };
 }
+
+// npp filters
+npp_filter = {
+    p:out_param_npp x:constant_Any => {
+        const op = x.getType().isString ? '=~' : '==';
+        return C.makeFilter($options, p, op, x);
+    };
+
+    p:out_param_npp x:constant_Any => {
+        return C.makeFilter($options, p, 'contains', x);
+    };
+}
+
+
+// pvp filters
+pvp_filter = {
+    p:out_param_pvp x:constant_Any => {
+        const op = x.getType().isString ? '=~' : '==';
+        return C.makeFilter($options, p, op, x);
+    };
+
+    p:out_param_pvp x:constant_Any => {
+        return C.makeFilter($options, p, 'contains', x);
+    };
+}
+
+// avp filters
+avp_filter = {
+    p:out_param_avp x:constant_Any => {
+        const op = x.getType().isString ? '=~' : '==';
+        return C.makeFilter($options, p, op, x);
+    };
+
+    p:out_param_avp x:constant_Any => {
+        return C.makeFilter($options, p, 'contains', x);
+    };
+}
+
+
+// npi filters
+npi_filter = {
+    ('that' | '') 'is' 'the' p:out_param_npi x:constant_Any => {
```

Comment:
should we include a variant with "are" instead of "is" (here and elsewhere)?
maybe for robustness/completeness and not for turking? or as a postprocessing?

Reply:
I thought about that. This will make the templates look way messier. I think postprocessing might be a better solution, which should include other grammar fixes, such as changing the pronoun and tense accordingly.